packages/auth/src/core/user/user_impl.ts

```diff
@@ -142,6 +142,8 @@ export class UserImpl implements UserInternal {
   _clone(auth: AuthInternal): UserInternal {
     return new UserImpl({
       ...this,
+      createdAt: this.metadata.creationTime,
```

Comment:
Can we define this in a `newUser` variable and use `newUser.metadata._copy(this.metadata);` instead?

Reply:
done.